profiles/coreos/targets/sdk/package.accept_keywords

```diff
@@ -11,3 +11,6 @@ virtual/rust ~amd64
 # Accept go utilities
 dev-go/glide ~amd64
 dev-go/godep ~amd64
+
+# Accept perl libs required to build >=samba-4.12
+dev-perl/Parse-Yapp ~amd64 ~arm64
```

Comment:
Is this needed? The `dev-perl/Parse-Yapp` package has keywords like `KEYWORDS="~alpha amd64 arm arm64 hppa …`, so shouldn't the build be fine without this entry?

Reply:
Yes, it's needed, or else the SDK build breaks. Core issue here is that Samba is a package required by the OS image while Parse-Yapp needs to be part of the SDK in order to build Samba. Gentoo is not able to map this dependency.

Reply:
(We were initially missing `~arm64` there which is why the ARM SDK build broke)